parsers/getter/tests/test_ust.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+from datetime import date, datetime
+import requests_mock
+from decimal import Decimal
+import bs4
+from parsers.getter.ust import (make_year,
+                                make_url,
+                                fetch,
+                                format_value,
+                                get_date,
+                                rename_variable,
+                                parse_xml,
+                                yield_ust_dict)
+
+#fixture
+
+def fake_content():
+    return """<?xml version="1.0" encoding="utf-8" standalone="yes"?><pre>
+                <m:properties>
+                <d:NEW_DATE>2017-01-03T00:00:00</d:NEW_DATE>
+                <d:BC_1MONTH>0.52</d:BC_1MONTH>"""
+
+def fake_fetch(url=None):
+    return """<?xml version="1.0" encoding="utf-8" standalone="yes"?><pre>
+                <m:properties>
+                <d:NEW_DATE>2017-01-03T00:00:00</d:NEW_DATE>
+                <d:BC_1MONTH>0.52</d:BC_1MONTH>"""
+
+class Test_make_year:
+    def test_make_year_with_good_date(self):
+        assert make_year(date(2017, 1, 1)) == 2017
+
+    def test_make_year_with_year_out_of_range_raises_ValueError(self):
+        with pytest.raises(ValueError):
+            make_year(date(1989, 1, 1))
+
+    def test_make_year_with_Non_date_parameter_raises_AttributeError(self):
+        with pytest.raises(AttributeError):
+            make_year(None)
+
+def test_make_url():
+    year = 1850
```

Comment:
This is obscuring: tests are meant to be straightforward, to reader as well. 1850 is invalid value, but tests pass because we test a different thing. Please change this to valid value. The test shoukd have one meaning, this one has sevrral.